README.md

```diff
@@ -178,8 +179,8 @@ app.use(
         );
       }
 
-      // Redirect to app with shop parameter upon auth
-      ctx.redirect(`/?shop=${shop}`);
+      // Redirect to app with shop and host parameter upon auth
+      ctx.redirect(`/?shop=${shop}&host=${host}`);
```

Comment:
@ryan-dutton do we need to add `host` here too? 

```
router.get("/", async (ctx) => {
  const shop = ctx.query.shop;

  // If this shop hasn't been seen yet, go through OAuth to create a session
  if (ACTIVE_SHOPIFY_SHOPS[shop] === undefined) {
    ctx.redirect(`/auth?shop=${shop}`); <===== &host=${host}
  } else {
    // Load app skeleton. Don't include sensitive information here!
    ctx.body = '🎉';
  }
});
```

Reply:
Thanks for looking. Based on the equivalent route in the example Shopify cli node app's server.js, it's not required.  The change I am suggesting certainly solved my issues.

Reply:
Also, I don't think the host parameter is guaranteed to be available at that point, it's seems to be something Shopify sends through specifically in the OAuth flow.

Reply:
I see.  I was not sure if that is needed.  Also, I do not see `host` in `ctx.query`.  It carries some some gen stuff.

Reply:
Fair enough, I'm certainly not claiming to be authoritive on the matter :)  I just noticed the difference in the process of trying to use this example as opposed to the NextJS [server.js](https://github.com/Shopify/shopify-app-node/blob/master/server/server.js) that the Shopify CLI generates.

Reply:
Actually, I think I found some supporting reasoning. There is a commit: [Use App Bridge V2](https://github.com/Shopify/shopify-app-node/commit/7430eca3d810adafa30565c3c6b2f10a14dd195b)